### PR TITLE
Adjust system limits to allow e2e tests to run correctly

### LINKED ISSUE #631: Multi E2E tests started failing in GHA
It appears that "something" changed in GHA which led to multiple consistent failures in the multi cluster e2e tests.
We need to stabilize them in order to get them passing again by adjusting system limits and reducing their pressure.

### PR DESCRIPTION
**Pull request by adejanovski**
**What this PR does**:
Fixes failing multi e2e tests due to the open file limit and watcher limit being set too low in GHA.
They started failing consistently on GHA a couple days ago while they were passing nicely on our laptops.

**Which issue(s) this PR fixes**:
Fixes #631

**Checklist**
- [x] Changes manually tested
- [x] Automated Tests added/updated
- [ ] Documentation added/updated
- [ ] CHANGELOG.md updated (not required for documentation PRs)
- [x] CLA Signed:  [DataStax CLA](https://cla.datastax.com/)

### CODE REVIEW

File: .github/workflows/kind_e2e_tests.yaml [hunk 1]
```diff
@@ -92,6 +92,12 @@ jobs:
       CONTROL_PLANE: kind-k8ssandra-0
       DATA_PLANES: kind-k8ssandra-0
     steps:
+      - name: Raise open files limit
+        run: |
+          echo fs.file-max=1000000 | sudo tee -a /etc/sysctl.conf && sudo sysctl -p
```
Comment (adutra) [inline .github/workflows/kind_e2e_tests.yaml:97]: Maybe you can do `sudo sysctl -p` just once after the 3 echo commands?

Comment (adejanovski) [reply .github/workflows/kind_e2e_tests.yaml:97]: right 👍

**Review by emerkle826 — APPROVED** [review]
This looks good to me. Just more of a question. I guess updating the `fs.file-max` in sysctl.conf is different than upping the open file limit in `ulimit`? The output of `ulimit -a` shows the open file limit to be 65K (65536):
```
open files                      (-n) 65536
```
Regardless, it seems the changes are getting the e2e tests to pass.

File: .github/workflows/kind_e2e_tests.yaml (hunk 1 above)
Comment (emerkle826) [reply .github/workflows/kind_e2e_tests.yaml:97]: @adejanovski Not sure if you wanted to make the suggested change here, but it looks like the sysctl.conf values are still dumped 3 times here.

**Comment by emerkle826** [comment]
@adejanovski All green!

**Comment by adejanovski** [comment]
> This looks good to me. Just more of a question. I guess updating the `fs.file-max` in sysctl.conf is different than upping the open file limit in `ulimit`? The output of `ulimit -a` shows the open file limit to be 65K (65536):
> 
> ```
> open files                      (-n) 65536
> ```
> 
> Regardless, it seems the changes are getting the e2e tests to pass.

Yeah, to my surprise, it is different.

**Pull request merged**